### PR TITLE
Add message if the autoupdater is disabled

### PR DESCRIPTION
**Pull request by rubo77**
If the Auto Updater is disabled, this PR adds a message:

### CODE REVIEW

**Comment by rubo77** [comment]
I tested this in v2017.1

File: package/gluon-config-mode-autoupdater/luasrc/lib/gluon/config-mode/wizard/0050-autoupdater-info.lua [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+	if enabled then
 		local s = form:section(
 			Section, nil,
 			translate('This node will automatically update its firmware when a new version is available.')
 		)
+	else
+		local s = form:section(
+			Section, nil,
+			translate('Automatic updates are disabled. They can be enabled in <em>Advanced settings</em>')
+		)
```
Comment (NeoRaider) [inline package/gluon-config-mode-autoupdater/luasrc/lib/gluon/config-mode/wizard/0050-autoupdater-info.lua]: Assigning the section to a local variable doesn't make sense here... while at it, you can also fix that for the branch you didn't touch.

Comment (rubo77) [inline package/gluon-config-mode-autoupdater/luasrc/lib/gluon/config-mode/wizard/0050-autoupdater-info.lua]: Ah i got what you meann with "branch"

done

Comment (rubo77) [inline package/gluon-config-mode-autoupdater/luasrc/lib/gluon/config-mode/wizard/0050-autoupdater-info.lua]: I removed those useless definitions globally here: https://github.com/freifunk-gluon/gluon/pull/1150

**Comment by rubo77** [comment]
tested and working.

File: package/gluon-config-mode-autoupdater/luasrc/lib/gluon/config-mode/wizard/0050-autoupdater-info.lua [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+	local enabled = uci:get_bool("autoupdater", "settings", "enabled")
+	if enabled then
+		form:section(
 			Section, nil,
 			translate('This node will automatically update its firmware when a new version is available.')
 		)
+	else
+		form:section(
+			Section, nil,
+			translate('Automatic updates are disabled. They can be enabled in <em>Advanced settings</em>')
```
Comment (NeoRaider) [inline package/gluon-config-mode-autoupdater/luasrc/lib/gluon/config-mode/wizard/0050-autoupdater-info.lua]: This sentence is missing a period at the end. (French version too)

Comment (rubo77) [inline package/gluon-config-mode-autoupdater/luasrc/lib/gluon/config-mode/wizard/0050-autoupdater-info.lua]: done

**Pull request merged**